bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -65,8 +68,21 @@ chi_eff_n = convert.chi_eff(mass1_n, mass2_n, spin1z_n, spin2z_n)
 mchirp_n = convert.mchirp_from_mass1_mass2(mass1_n, mass2_n)
 eta_n = convert.eta_from_mass1_mass2(mass1_n, mass2_n)
 
+assert len(args.fit_param) == len(args.log_param)
+
 if args.make_template_kde:
-    sample_n = numpy.vstack((numpy.log(mchirp_n), eta_n, chi_eff_n)).T
+    param0 = eval(args.fit_param[0][:])
```

Comment:
As we discussed, these lines can be replaced by a loop over the list of fit param names from the command line. 

Also I think the template parameters have to be evaluated for all cases, or I think it's a good idea to do so, so the evaluation can occur outside the 'if' statement